my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language is the [Fuzzy Control Language](http://ffll.sourceforge.net/fcl.htm).
 
 # Domain
 _Describe the language's domain in five words._
 
+Fuzzy logic instead of booleans.
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a ContextFree program runs._
 
+In simple terms, when a program written in a domain-specific language is run, its purpose is to generate specific code that can be integrated into the application that implements it. In the case of FCL, this would involve the generation of a set of rules and categories beyond the scope of booleans that could be interpreted by its parent application. For ContextFree, a specific set of terms (eg. `CIRCLE`) are being transformed into code that the parent application is capable of parsing. In either case, the language is an interface between the user and the parent application that enables the user to perform a specific task within the greater context of that application. 
```

Comment:
Can you expand on what you mean by "rules and categories beyond the scope of booleans"? I also don't know what language this works in or if it is a standalone DSL.